src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -23,9 +35,48 @@ public class CommandPat
         private const string ScopeOption = "--scope";
         private const string ScopeHelp = "A token scope for accessing Azure DevOps resources. Repeated invocations allowed.";
 
+        private const string PromptHintOption = "--prompt-hint";
+        private const string PromptHintHelp = "A prompt hint to contextualize prompts and identify uses in telemetry, when captured.";
+
         private const string OutputOption = "--output";
         private const string OutputHelp = "How PAT information is displayed. [default: token]\n[possible values: none, status, token, base64, header, headervalue, json]";
 
+        private const string ModeOption = "--mode";
+#if PlatformWindows
+        /// <summary>
+        /// The help text for the <see cref="ModeOption"/> option.
+        /// </summary>
+        private const string ModeHelp = @"Authentication mode. Repeated invocations allowed. [default: iwa (Integrated Windows Auth), then broker, then web]
+[possible values: all, iwa, broker, web, devicecode]";

```

Comment:
```suggestion
        private const string ModeHelp = @"Authentication mode. Repeated invocations allowed. 
[default: iwa (Integrated Windows Auth), then broker, then web]
[possible values: all, iwa, broker, web, devicecode]";
```

Reply:
Newlines are preserved in the help text rendering, and all of it is auto-wrapper on the CLI based on terminal width. So I generally try to use newlines liberally to make reading the help easier.